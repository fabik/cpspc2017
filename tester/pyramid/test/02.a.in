2 50
30995353 2167368079
10629428449
12554553645
10880938871
142708043
16415811245
13466849494
11631790449
30927216712
6037243503
24973025576
19635634954
21391785178
23598829579
8841964638
9217427147
7399374933
18422743496
23016455431
21525216681
7286948595
22213229797
20286911680
20270273636
1316235943
29399133659
27613777124
9832612611
20595495161
16487011483
975113883
19312979383
25106096647
14135934724
27777048236
17581544197
22218673060
32365514537
19938555640
865049884
3474372554
28999539476
17148779195
30951567048
7201521900
11212282517
22473856239
11651719410
30395266918
4210652235
29733473267
