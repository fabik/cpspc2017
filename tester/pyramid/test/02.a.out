324817695889521830
383646327513736554
332503437054396495
4360921007864586
501639952766856944
411524574908976797
355448215407602549
945084424942887181
184488143791568622
763134223654208281
600032422613925385
653697459467701371
721141074210540233
270195767801334133
281669279063125001
226112620116115767
562968472247634470
703344686587414816
657774904787856913
222677057778204696
678799444013093185
619934358723219808
619425927690098428
40221986388529174
898388742400786406
843831209142158801
300468326061841518
629364157847983730
503815713879348789
29797862259637675
590172603756171011
767200654775675273
431970709293963715
848820503404723219
537262817464925647
678965781101888686
989036419883272643
609289176170667251
26434489059719547
106171060195464258
886177807243789336
524038237094396922
945828530816293123
220066559718953976
342628749087900609
686763755634097173
356057211373290500
928828921887564082
128670545528372650
908605606059349033
